frontend/public/components/persistent-volume-claim.jsx

```diff
@@ -8,8 +8,8 @@ import { ResourceEventStream } from './events';
 
 const pvcPhase = pvc => pvc.status.phase;
 
-const { common } = Kebab.factory;
-const menuActions = [...common];
+const { common, ExpandPVC } = Kebab.factory;
+const menuActions = [...common, ExpandPVC];
```

Comment:
I think this is more consistent with other menus:

```suggestion
const menuActions = [ExpandPVC, ...common];
```